src/Scheduler.ts

```diff
@@ -17,7 +21,7 @@ import { Subscription } from './Subscription';
  *
  * @class Scheduler
  */
-export class Scheduler {
+export class Scheduler implements IScheduler {
 
   public static now: () => number = Date.now ? Date.now : () => +new Date();
```

Comment:
@blesh from what I recall, it's static to mirror the [Rx4 Scheduler API](https://github.com/Reactive-Extensions/RxJS/blob/master/src/core/concurrency/scheduler.js#L45)